src/Snooze/Balance/Http.hs

```diff
@@ -29,3 +30,14 @@ httpBalanced req = ReaderT $ \(BalanceConfig ubt rp mgr) -> EitherT $ do
   liftIO . fmap (\(m, e) -> maybeToRight (BalanceTimeout e) m) $ randomRoundRobin' rp
     (\_ b -> catch (fmap Right . httpGo mgr . req . balanceRequest $ b) (\(e :: HttpException) -> pure $ Left e))
     (balanceTableList bt)
+
+httpBalancedReq ::
+    (Applicative m, MonadIO m)
+  => Request
+  -> BalanceT m (Response BSL.ByteString)
+httpBalancedReq r = ReaderT $ \(BalanceConfig ubt rp mgr) -> EitherT $ do
```

Comment:
If we're going with this new function do you mind creating an extra `httpBalancedReq'` that is shared between the two function? Don't really want this logic duplicated.